dev_docs/key_concepts/data_views.mdx

```diff
@@ -27,3 +27,10 @@ Users can create data views via [Data view management](https://www.elastic.co/gu
 Additionally, they can be created through the data view API.
 
 Data views also allow formatters and custom labels to be defined for fields.
+
+### Services
+
+**hasData:** A standardized way to check the empty state for indices and data views.
+- `hasESData: () => Promise<boolean>; // Check to see if ES data exists`
+- `hasDataView: () => Promise<boolean>; // Check to see if any data view exists (primitive or user created)`
```

Comment:
>(primitive or user created)

I think `managed or user created` would be a better description

Reply:
done